src/sonification/index.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+var c2m = require('chart2music');
+var Fx = require('../components/fx');
+var Lib = require('../lib');
+
+var codecs = require('./all_codecs').codecs;
+
+/* initClosedCaptionDiv: Initialize the closed caption div with the given configuration.
+ * This function works by either creating a new div or returning the existing div
+*/
+function initClosedCaptionDiv(gd, config) {
+    if(config.generate) {
+        var closedCaptions = document.createElement('div');
+        closedCaptions.id = config.elId;
+        closedCaptions.className = config.elClassname;
+        gd.parentNode.insertBefore(closedCaptions, gd.nextSibling); // this really might not work
+        return closedCaptions;
+    } else {
+        return document.getElementById(config.elId);
+    }
```

Comment:
We could simply return and drop else.
```suggestion
    }
    return document.getElementById(config.elId);
```